R/plotly_IMAGE.R

```diff
@@ -9,7 +9,8 @@
 #' @param format The desired image format 'png', 'jpeg', 'svg', 'pdf', 'eps', or 'webp'
 #' @param scale Both png and jpeg formats will be scaled beyond the specified width and height by this number.
 #' @param out_file A filename for writing the image to a file.
-#' @param ... arguments passed onto `httr::POST`
+#' @param ...  arguments passed onto `httr::RETRY`
+#' @importFrom httr RETRY write_disk
```

Comment:
Please remove this line as it's also in `R/imports.R`

Reply:
oh I see, my mistake. I've not seen that pattern of putting all `importFrom`s in one place before. I'll remove this